rasa_core/tracker_store.py

```diff
@@ -39,6 +40,11 @@ def find_tracker_store(domain, store=None, event_broker=None):
                                      host=store.url,
                                      event_broker=event_broker,
                                      **store.kwargs)
+        elif store.type == 'SQL':
+            return SQLTrackerStore(domain=domain,
+                                   host=store.url,
```

Comment:
well either use url or host, according to docs the user would supply `store.host`